File: app-java/src/main/java/com/TritiumGaming/phasmophobiaevidencepicker/firebase/firestore/transactions/store/merchandise/FirestoreMerchandise.java
package com.TritiumGaming.phasmophobiaevidencepicker.firebase.firestore.transactions.store.merchandise;

import com.TritiumGaming.phasmophobiaevidencepicker.firebase.firestore.transactions.store.FirestoreMarketplace;
import com.google.firebase.firestore.CollectionReference;
import com.google.firebase.firestore.DocumentReference;

public class FirestoreMerchandise {

    private static final String
            DOCUMENT_MERCHANDISE = "Merchandise";

    public static DocumentReference getMerchandiseDocument() throws Exception {
        return FirestoreMarketplace.getStoreCollection().document(DOCUMENT_MERCHANDISE);
    }

}

File: app-compose/src/main/java/com/tritiumgaming/phasmophobiaevidencepicker/data/firebase/firestore/transactions/store/merchandise/FirestoreMerchandise.kt
package com.tritiumgaming.phasmophobiaevidencepicker.data.firebase.firestore.transactions.store.merchandise

import com.google.firebase.firestore.DocumentReference
import com.tritiumgaming.phasmophobiaevidencepicker.data.firebase.firestore.transactions.store.FirestoreMarketplace.Companion.storeCollection

class FirestoreMerchandise {

    companion object {
        private const val DOCUMENT_MERCHANDISE = "Merchandise"

        val merchandiseDocument: DocumentReference
            get() = storeCollection.document(DOCUMENT_MERCHANDISE)
    }
}
